develop/components/pages/ModalPage.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+import { Modal } from "../../../src/Modal/Modal";
+import { Button } from "../../../src/Button/Button";
+import { getParameterByName } from "../../utils/queryString";
+const Highlight = (require("react-highlight")).default;
+const docMD = require("../../../src/Modal/readme.md");
+
+export default class ModalPage extends React.Component<any, any>  {
```

Comment:
I noticed we have a state of type any for all the component pages despite the fact that have states in all the components. I recommend we should change all of them starting with this, because this project being open source means the code quality is open to criticism.

Reply:
I dont think we can use `ModalProps` for ModalPage, 
if you check line 29 contains `this.props.location` that has nothing to do with `ModalProps`
I also dont believe that props of modal and props of a page are similar.

Reply:
No @omarVodiaK i am not saying they are similar and also what i am trying to point out is nit just peculiar to this PR. I am suggesting we should perhaps to it do the remaining page so here

```
interface ModalPageState {
  dialogue: boolean;
  fullscreen: boolean;
  position: string;
  disableBackdropDismiss: boolean;
} 
```

and 

export default class ModalPage extends React.Component<any, ModalPageState >  {